assets/jsx/bagger.jsx

```diff
@@ -19,15 +19,11 @@ class Bagger extends React.Component {
     componentDidMount() {
         const { dispatch } = this.props
         const b = document.getElementById('bagger');
-        if (b.dataset.accessKeyId !== undefined) { // TODO: any? all? add fine grain actions?
-            const dataset = b.dataset
-            dispatch(BagActions.updateConfig(
-                dataset.accessKeyId,
-                dataset.secretAccessKey,
-                dataset.bucket,
-                dataset.region,
-                dataset.keyPrefix
-                ))
+        const dataset = b.dataset
+        const args = [dataset.accessKeyId, dataset.secretAccessKey, dataset.bucket, dataset.region,
+                        dataset.keyPrefix]
```

Comment:
https://www.npmjs.com/package/esformatter-jsx definitely works but it consistently applies a style which I'm not entirely in love with:

https://gist.github.com/acdha/783491c31c10a41d4ab0